packages/flutter/test/material/time_picker_test.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 @TestOn('!chrome')
+import 'dart:async';
```

Comment:
I don't think this import is necessary. What was the reason for adding it?

Reply:
There is no reason. I think I was writing a test that required on it but found a better way to do it but forgot to remove the import. I'll get that removed